docs/install.rst

```diff
@@ -108,6 +110,13 @@ The following packages can optionally be used when testing:
 Installing ``astropy``
 ======================
 
+If you are new to Python and/or do not have familiarity with `Python virtual
```

Comment:
I'm thinking of the new user that just comes to this page wanting to install astropy.  What they'll likely run is the first command they see, which means opening a terminal window and typing `pip install astropy`.  That will generally fail => BAD instructions.